app/models/alchemy/page.rb

```diff
@@ -350,41 +333,66 @@ def publish!
     # @param [TreeNode]
     #   A tree node with new lft, rgt, depth, url, parent_id and restricted indexes to be updated
     #
-    def update_node!(node)
-      hash = {lft: node.left, rgt: node.right, parent_id: node.parent, depth: node.depth, restricted: node.restricted}
+    # def update_node!(node)
+    #   hash = {lft: node.left, rgt: node.right, parent_id: node.parent, depth: node.depth, restricted: node.restricted}
```

Comment:
Metrics/LineLength: Line is too long. [120/100] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)